client/my-sites/theme/main.jsx

```diff
@@ -339,9 +342,20 @@ class ThemeSheet extends Component {
 		return preview.action( this.props.themeId );
 	};
 
+	shouldRenderForStaging() {
+		// isExternallyManagedTheme determines if a theme is paid or not
+		const { isExternallyManagedTheme, isWpcomStaging } = this.props;
+		return isExternallyManagedTheme && isWpcomStaging;
```

Comment:
We've checked the externallyManagedThemes and all of them are paid, but I believe this assumption could change.
Is possible WPcom adds a third-party theme that is free?

Reply:
Probably, but this is used everywhere that we check if a theme is paid. So I guess, this needs to be changed everywhere